packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -0,0 +1,254 @@
+import { LoggerProxy, MessageEventBusDestinationOptions } from 'n8n-workflow';
+import { DeleteResult } from 'typeorm';
+import { EventMessageTypes } from '../EventMessageClasses/';
+import { MessageEventBusDestination } from '../MessageEventBusDestination/MessageEventBusDestination.ee';
+import { MessageEventBusLogWriter } from '../MessageEventBusWriter/MessageEventBusLogWriter';
+import EventEmitter from 'node:events';
+import config from '@/config';
+import { Db } from '@/index';
```

Comment:
To avoid the circular dependency

```suggestion
import * as Db from '@/Db';
```